main.nf

```diff
@@ -5,5 +5,11 @@ nextflow.enable.dsl = 2
 include { HELLO } from "./modules/hello"
```

Comment:
Would not name processes/modules/files as hello/foo/bar/baz  and the sort of

Reply:
what would you suggest? This is still a demo process that is to show the others how a module is called with the input channels and will be removed once we get DESeq2 in!

Reply:
maybe a bit more descriptive:
dummyCheckInput

Reply:
sounds good, done!